examples/solids/elasticity.c

```diff
@@ -844,17 +901,16 @@ int main(int argc, char **argv) {
   // ---------------------------------------------------------------------------
   // Compute energy
   // ---------------------------------------------------------------------------
+  PetscReal energy;
+  ierr = ComputeStrainEnergy(dm_energy, res_ctx, ceed_data[fine_level]->op_energy,
+                             U, &energy); CHKERRQ(ierr);
   if (!app_ctx->test_mode) {
-    // -- Compute L2 error
-    CeedScalar energy;
-    ierr = ComputeStrainEnergy(dm_energy, res_ctx, ceed_data[fine_level]->op_energy,
-                               U, &energy); CHKERRQ(ierr);
-
     // -- Output
     ierr = PetscPrintf(comm,
                        "    Strain Energy                      : %.12e\n",
                        energy); CHKERRQ(ierr);
   }
+  ierr = RegressionTests_solids(app_ctx, energy); CHKERRQ(ierr);
```

Comment:
Do we want this to always run or only run in `test_mode`?

Reply:
Yeah, if the user passes `-expect_final_strain_energy` then we should test it, regardless of "test" mode (which roughly means "quiet", but also adjusting defaults).